File: src/test/java/cernx/colt/matrix/tfloat/impl/SparseCCMFloatMatrix2DViewTest.java
package cernx.colt.matrix.tfloat.impl;

public class SparseCCMFloatMatrix2DViewTest extends SparseCCMFloatMatrix2DTest {

    public SparseCCMFloatMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCMFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseCCMFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseCCMFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}

File: src/test/scala/cern/colt/matrix/tfloat/impl/SparseCCMFloatMatrix2DViewTest.scala
package cern.colt.matrix.tfloat.impl

import cern.colt.matrix.MatrixTypes.SparseCCMFloatMatrix2D

class SparseCCMFloatMatrix2DViewTest(arg0: String) extends SparseCCMFloatMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseCCMFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseCCMFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseCCMFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
